### PR TITLE
Cleanup: Improve Connection Handling in Profile.

### PR DESCRIPTION
**Pull request by mikeller**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [x] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Changes made:

1. improve naming;
2. remove unneeded disconnects.

### CODE REVIEW

File: profile-widget/profilewidget2.cpp [hunk 1]
```diff
@@ -480,7 +469,6 @@ void ProfileWidget2::setPlanState(const dive *d, int dc)
 	setProfileState(d, dc);
 	mouseFollowerHorizontal->setVisible(true);
 	mouseFollowerVertical->setVisible(true);
-	disconnectTemporaryConnections();
```
Comment (bstoeger) [inline profile-widget/profilewidget2.cpp:483]: Have you checked if this may lead to double connections? Those can be very painful.

Comment (mikeller) [reply profile-widget/profilewidget2.cpp:483]: Yes - `setProfileState()` right above already calls `disconnectTemporaryConnections()`.

Comment (bstoeger) [reply profile-widget/profilewidget2.cpp:483]: Ah, ok.

This obviously collides with my efforts to port the profile to QtQuick, but oh weill...

Comment (mikeller) [reply profile-widget/profilewidget2.cpp:483]: Apologies for the collision. I am on a quest to make the addition of gases and gas switches work for manually added dives, and in the process I have been cleaning up some less than ideal implementation details I've come across.

**Pull request merged**